### ISSUE TITLE
FATAL: except not rethrown

### ISSUE THREAD

**Issue by ryanleecode** [livekit/agents#732]
no idea whats causing this

version: 0.8.10

```
{"message": "livekit::rtc_engine:611:livekit::rtc_engine - resuming connection... attempt: 0", "level": "ERROR", "pid": 581437, "job_id": "AJ_xUzDVbPtmW82", "timestamp": "2024-09-10T03:10:31.785861+00:00"}
{"message": "livekit::rtc_engine::peer_transport:150:livekit::rtc_engine::peer_transport - peer connection is closed, cannot create offer", "level": "WARNING", "pid": 581437, "job_id": "AJ_xUzDVbPtmW82", "timestamp": "2024-09-10T0:10:32.064398+00:00"}
{"message": "livekit::rtc_engine:613:livekit::rtc_engine - resuming connection failed: connection error: closed", "level": "ERROR", "pid": 581437, "job_id": "AJ_xUzDVbPtmW82", "timestamp": "2024-09-10T03:10:32.067442+00:00"}
{"message": "livekit::rtc_engine:545:livekit::rtc_engine - failed to reconnect to the livekit room", "level": "ERROR", "pid": 581437, "job_id": "AJ_xUzDVbPtmW82", "timestamp": "2024-09-10T03:10:32.067975+00:00"}
FATAL: exception not rethrown
```

**Comment by theomonnom** [comment]
the `FATAL: exception not rethrown` is an issue we know internally, it is completely harmless and only happens when the child process ends. I'm going to work on a fix for the end of thing month (not a huge priority since there shouldn't be any side effects)

**Comment by theomonnom** [comment]
Hey, this should now be fixed with `livekit==0.17.3`
Please reopen if you're still seeing this issue
